subsys/nrf_security/src/drivers/cracen/cracenpsa/src/key_management.c

```diff
@@ -637,11 +637,15 @@ static psa_status_t export_ecc_public_key_from_keypair(const psa_key_attributes_
 		if (key_buffer_size != sizeof(ikg_opaque_key)) {
 			return PSA_ERROR_INVALID_ARGUMENT;
 		}
-		priv_key =
-			si_sig_fetch_ikprivkey(sx_curve, ((ikg_opaque_key *)key_buffer)->owner_id);
-		data[0] = SI_ECC_PUBKEY_UNCOMPRESSED;
-		pub_key.key.eckey.qx = &data[1];
-		pub_key.key.eckey.qy = &data[1 + sx_pk_curve_opsize(sx_curve)];
+
+		if (IS_ENABLED(PSA_NEED_CRACEN_ECDSA_SECP_R1_256)) {
```

Comment:
Minor: How about using an `#ifdef` instead? (Seems more clear to me when editor syntax highlighting works according to the configuration.)

Reply:
Will not do that, we decided to use the IS_ENABLED so time before all around the cracen driver. So first of all I want to be consistent. But also it is more easily readable than ifdef in my opinion and it allows you to catch build failures at build time even if the condition is not enabled.

Reply:
It seems that it's actually a bit of a mix (there are some `#ifdef`s)... But fine. :slightly_frowning_face: